app/networkapi/settings.py

```diff
@@ -53,11 +55,18 @@
 
 SITE_ID = 1
 
+# Use social authentication if there are key/secret values defined
+SOCIAL_AUTH_GOOGLE_OAUTH2_KEY = env('SOCIAL_AUTH_GOOGLE_OAUTH2_KEY')
+SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET = env('SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET')
+SOCIAL_SIGNIN = SOCIAL_AUTH_GOOGLE_OAUTH2_KEY is not None and \
+                    SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET is not None
+
 # Application definition
 
-INSTALLED_APPS = [
+INSTALLED_APPS = list(filter(None, [
```

Comment:
Correct - the conditional may yield a `None`, which needs to be filtered out because Django will crash hard if there are empty strings or `None` entries in here.